cookiecutter.json

```diff
@@ -1,4 +1,5 @@
 {
+  "year": "{{ cookiecutter._current_year }}",
   "full_name": "FernLab",
   "email": "fernlab@gfz-potsdam.de",
```

Comment:
When trying to create a package I now get an error:
`cookiecutter cookiecutter-pypackage
Unable to render variable 'year'
Error message: 'collections.OrderedDict object' has no attribute '_current_year'
`